src/OrchardCore/OrchardCore.OpenId.Core/YesSql/Stores/OpenIdAuthorizationStore.cs

```diff
@@ -340,6 +335,138 @@ public virtual async ValueTask<long> PruneAsync(DateTimeOffset threshold, Cancel
         return result;
     }
 
+    /// <inheritdoc/>
+    public virtual async ValueTask<long> RevokeAsync(
+        string subject, string client, string status, string type, CancellationToken cancellationToken)
+    {
+        Expression<Func<OpenIdAuthorizationIndex, bool>> query = index => true;
```

Comment:
Not an absolute fan of creating expression trees "manually", but AFAIK, YesSql still doesn't support `IQueryable<T>`, so this is likely the least horrible option 😅